<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1392</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imgception</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/APPICONS/icons/hd-icon.png</normaloff>:/APPICONS/icons/hd-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 112, 168);
background-color: rgb(64, 105, 159);</string>
  </property>
  <widget class="QLabel" name="ImageViewer_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>1081</width>
     <height>741</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="New_btn">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Refresh_btn">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Edit_btn">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="logout_btn">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>120</y>
     <width>251</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <column>
       <property name="text">
        <string notr="true">ME</string>
       </property>
       <property name="background">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/APPICONS/icons/world_wide_globe_online_business_web-128.png</normaloff>:/APPICONS/icons/world_wide_globe_online_business_web-128.png</iconset>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="CounterViewer_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>120</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>251</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="AddUser_btn">
        <property name="styleSheet">
         <string notr="true">color: rgb(144, 170, 255);
</string>
        </property>
        <property name="text">
         <string>Add user</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Update_image_data">
        <property name="styleSheet">
         <string notr="true">color: rgb(144, 170, 255);</string>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteUser_btn">
        <property name="styleSheet">
         <string notr="true">color: rgb(144, 170, 255);</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl_log">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
